src/test/java/fr/lunatech/timekeeper/services/UserEventServiceTest.java

```diff
@@ -1,15 +1,15 @@
 package fr.lunatech.timekeeper.services;
 
 import fr.lunatech.timekeeper.models.time.UserEvent;
+import fr.lunatech.timekeeper.timeutils.TimeKeeperDateUtils;
 import org.junit.jupiter.api.Test;
-
 import java.time.LocalDateTime;
-
 import static org.junit.jupiter.api.Assertions.*;
 
+
 class UserEventServiceTest {
 
-    private final UserEventService userEventService = new UserEventService();
+    UserEventService userEventService = new UserEventService();
```

Comment:
You should maybe create a local instance in each test. We're not sure of what the constructor does